packages/video_player/video_player/example/ios/Runner.xcodeproj/project.pbxproj

```diff
@@ -274,6 +274,7 @@
 				TargetAttributes = {
 					97C146ED1CF9000F007C117D = {
 						CreatedOnToolsVersion = 7.3.1;
+						DevelopmentTeam = S8QB4VV633;
```

Comment:
None of the changes in this file should be checked in either.

Reply:
done